src/allotropy/parsers/appbio_quantstudio_designandanalysis/structure/generic/structure.py

```diff
@@ -132,6 +132,12 @@ def get_result_class(cls) -> type[Result]:
     def __hash__(self) -> int:
         return hash(self.identifier)
 
+    @classmethod
+    def get_amplification_data_sheet(
```

Comment:
probably a q for another time but is there a reason we dont have `contents: DesignQuantstudioContents` as a property of objects of this `WellItem` class or said another way is there a reason this is a class method? it seems like more of an object method flavour at a glance

Reply:
I think this smells like over-design. Why does this method exist instead of just calling `contents.get_non_empty_sheet("Amplification Data")`?

Reply:
Up to this point the experiment types were handled with if statements, now the idea is to have a generic structure and when an experiment type needs a different treatment it can overwrite the default behavior. That is the reason this is a classmethod and that is the reason why the method is so simple in this case.